.env

```
@@ -10,13 +10,16 @@ COLLECTOR_CONTRIB_IMAGE=otel/opentelemetry-collector-contrib:0.108.0
 FLAGD_IMAGE=ghcr.io/open-feature/flagd:v0.11.2
 GRAFANA_IMAGE=grafana/grafana:11.2.0
 JAEGERTRACING_IMAGE=jaegertracing/all-in-one:1.60
-# must also update version field in ./src/grafana/provisioning/datasources/opensearch.yaml
-OPENSEARCH_IMAGE=opensearchproject/opensearch:2.16.0
+OPENSEARCH_IMAGE_VERSION=2.16.0
+OPENSEARCH_IMAGE=opensearchproject/opensearch:${OPENSEARCH_IMAGE_VERSION}
 POSTGRES_IMAGE=postgres:16.4
 PROMETHEUS_IMAGE=quay.io/prometheus/prometheus:v2.54.1
-VALKEY_IMAGE=valkey/valkey:8.0-alpine
-# must also update the version arg in ./test/tracetesting/Dockerfile
-TRACETEST_IMAGE=kubeshop/tracetest:v1.5.2
+TRACETEST_IMAGE_VERSION=v1.5.4
+TRACETEST_IMAGE=kubeshop/tracetest:${TRACETEST_IMAGE_VERSION}
+VALKEY_IMAGE=valkey/valkey:7.2-alpine
```

Is there any reason behind this version change?

```suggestion
VALKEY_IMAGE=valkey/valkey:8.0-alpine
```

---

@rogercoll I've reverted the update because it seems `8.0` is `RC`.
The latest stable is `7.2`.